nixos/modules/services/web-servers/apache-httpd/mediawiki.nix

```diff
@@ -91,12 +91,16 @@ let
     };
 
     skins = config.skins;
+    extensions = config.extensions;
 
     buildPhase =
       ''
         for skin in $skins; do
           cp -prvd $skin/* skins/
         done
+        for extension in $extensions; do
+          cp -prvd $extension/* extensions/
```

Comment:
It might be better, but I'm not familiar with mediawiki and don't know for certain that it'd accept symlinks. It might try to write to them.

As to the /*, note that $extension isn't a single extension; it's a directory containing extensions, which need to be copied including the directory name. That works the same way as for $skin, but if I changed it -- including by using a per-extension derivation -- I'd then need to pass the name of the extension out-of-band somehow.

Which might be fine if we had a complete list of them, but we don't. Today the way to add extensions to MediaWiki in NixOS is for the user to download them and put them in a directory... and that's probably fine.